src/prommis/solvent_extraction/solvent_extraction.py

```diff
@@ -1,5 +1,87 @@
+"""
+Solvent Extraction Model
+
+========================
+
+Author: Arkoprabho Dasgupta
+
+The Solvent Extraction unit model is used to perform the solvent extraction unit operation.
+It represents a series of tanks, referred to as stages, through which the aqueous and organic
+phases are passed through, and then subsequent extraction of the desired components happen.
+
+This model is based on the MSContactor model, but it only accounts for the material balance.
+Any kind of energy balance, pressure balance and reaction kinetics is not considered in this 
+model. However since this model is based on MSContactor, the energy balance and pressure balances 
```

Comment:
This is not correct. Whilst the default setting is False, the model does support energy and momentum balances for each stream (it does not support inter-stream heat transfer however).

Reply:
So I can remove the energy balance and pressure balance line, then. Will that be okay? I will keep the point that the configurations have to be set False in the stream config.

Reply:
You are confusing two separate things here. Your example property package does not include temperature and pressure, and thus for your examples you need to set these terms to False.

However, the SX model itself fully supports energy and momentum balances (as long as the property package has the necessary terms). Thus, this statement is incorrect. The model does support energy and momentum balances, and there is no need to set the arguments to False.

Reply:
Okay, I got your point. I will remove this term that you said